sunbeam-python/tests/unit/sunbeam/storage/backends/conftest.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: 2025 - Canonical Ltd
+# SPDX-License-Identifier: Apache-2.0
+
+"""Common fixtures and utilities for backend-specific tests."""
+
+import pytest
+
+from sunbeam.storage.backends.dellsc.backend import DellSCBackend
+from sunbeam.storage.backends.hitachi.backend import HitachiBackend
+from sunbeam.storage.backends.purestorage.backend import PureStorageBackend
+
+
+@pytest.fixture
+def hitachi_backend():
+    """Provide a Hitachi backend instance."""
+    return HitachiBackend()
+
+
+@pytest.fixture
+def purestorage_backend():
+    """Provide a Pure Storage backend instance."""
+    return PureStorageBackend()
+
+
+@pytest.fixture
+def dellsc_backend():
```

Comment:
I will note that the dellsc backend FC and iSCSI backends [are unsupported[(https://docs.openstack.org/cinder/latest/drivers.html#scfcdriver-unsupported) in the upstream as they don't have voting CI tests. We should potentially find a way to mark that this is an unsupported driver. Unsupported drivers risk being removed in the upstream.